docs/reference/aggregations.asciidoc

```diff
@@ -3,117 +3,367 @@
 
 [partintro]
 --
-The aggregations framework helps provide aggregated data based on a search query. It is based on simple building blocks
-called aggregations, that can be composed in order to build complex summaries of the data.
+An aggregation summarizes your data as metrics, statistics, or other analytics.
+Aggregations help you answer questions like:
 
-An aggregation can be seen as a _unit-of-work_ that builds analytic information over a set of documents. The context of
-the execution defines what this document set is (e.g. a top-level aggregation executes within the context of the executed
-query/filters of the search request).
+* What's the average load time for my website?
+* Who are my most valuable customers based on transaction volume?
+* What would be considered a large file on my network?
+* How many products are in each product category?
 
-There are many different types of aggregations, each with its own purpose and output. To better understand these types,
-it is often easier to break them into four main families:
+{es} organizes aggregations into four categories:
 
-<<search-aggregations-bucket, _Bucketing_>>::
-				A family of aggregations that build buckets, where each bucket is associated with a _key_ and a document
-				criterion. When the aggregation is executed, all the buckets criteria are evaluated on every document in
-				the context and when a criterion matches, the document is considered to "fall in" the relevant bucket.
-				By the end of the aggregation process, we'll end up with a list of buckets - each one with a set of
-				documents that "belong" to it.
+* <<search-aggregations-bucket,Bucket>> aggregations that
+group documents into buckets, also called bins, based on field values, ranges,
+or other criteria.
 
-<<search-aggregations-metrics, _Metric_>>::
-				Aggregations that keep track and compute metrics over a set of documents.
+* <<search-aggregations-matrix,Matrix>> aggregations experimental:[] that output
```

Comment:
We actually said a while back that we think we can remove "experimental" on matrix: https://github.com/elastic/elasticsearch/issues/53065

So you can probably drop this.